### ISSUE TITLE
@Quasar/Extras - MDI 7 has been updated to 7.2.96, but the CSS appears to be missing some icons

### ISSUE THREAD

**Issue by justinross** [quasarframework/quasar#16543]
### What happened?

Trying to use icons that have been added to the webfont after 7.0.96 results in the icons being blank. 

### What did you expect to happen?

The designated icons should display.

### Reproduction URL

https://stackblitz.com/edit/quasarframework-3k7xuj?file=src%2Fpages%2FIndexPage.vue

### How to reproduce?

Open the provided reproduction link and see that icons aren't displaying properly, but can be made to display by manually adding the appropriate CSS (which shows that they do exist in the webfont). 

### Flavour

Quasar CLI with Vite (@quasar/cli | @quasar/app-vite)

### Areas

Quasar Extras (@quasar/extras)

### Platforms/Browsers

Firefox

### Additional context

Basically, it looks like most of the files for the MDI7 webfont have been updated to 7.2.96, but the associated CSS file hasn't been updated since 7.0.96, so it's missing a couple hundred icons that have been added since then.

**Comment by rstoenescu** [comment]
Hi,

Thank you for opening this ticket.
This was a problem with multiple icon sets:
* mdi-v7
* bootstrap-icons
* eva-icons
* line-awesome
* themify

Fix will be available shortly in quasar/extras 1.16.8
